src/main/java/hu/wurfel/refference/school/diary/DiaryController.java

```diff
@@ -9,27 +9,28 @@
 import java.util.List;
 
 @RestController
+@RequestMapping("/api/v1/Diary")
 public class DiaryController {
 
 
 	DiaryService diaryService;
 
-	@GetMapping("/api/v1/Diary/search/{searchWith}/{searchBy}/{value}")
+	@GetMapping("/search/{searchWith}/{searchBy}/{value}")
 	public ResponseEntity<List<Diary>> search(@PathVariable EntityNames searchWith, @PathVariable EntityFieldNames searchBy, @PathVariable String value) {
 		return ResponseEntity.ok(diaryService.getSearchResponseList(searchWith, searchBy, value));
 	}
 
-	@PostMapping("/api/v1/Diary/adding")
+	@PostMapping("/adding")
```

Comment:
```suggestion
	@PostMapping
```

Reply:
már egyszer kivettem az adding, és a modify után a teljes URL-t, illetve a delete-nél csak a "/{id}" maradt meg. A searchnél még ottmaradt a search, azt elfelejtettem hogy azt is ki lehet venni. És bocsi, csak azt nem tudom hogy hogy működik pontosan a merge request. A request utáni commitok is belekerülnek, ha ugyan azon a branch-en vannak, vagy a requestnél csinál egy vágást, és az utána commitolt dolgokra uj merge request kell? De végigolvasom a suggessionöket és elfogadom őket, köszönöm